internal/rpc/rpcserver/server.go

```diff
@@ -3380,3 +3380,62 @@ func (s *walletServer) GetCoinjoinOutputspByAcct(ctx context.Context, req *pb.Ge
 		Data: resp,
 	}, nil
 }
+
+func (s *walletServer) SetAccountPassphrase(ctx context.Context, req *pb.SetAccountPassphraseRequest) (
+	*pb.SetAccountPassphraseResponse, error) {
+	defer zero(req.Passphrase)
+
+	_, err := s.wallet.AccountName(ctx, req.AccountNumber)
+	if err != nil {
+		if errors.Is(err, errors.NotExist) {
+			return nil, status.Errorf(codes.NotFound, "account not found")
+		}
+		return nil, err
+	}
+
+	lock := make(chan time.Time, 1)
+	defer func() {
+		lock <- time.Time{} // send matters, not the value
+	}()
+	err = s.wallet.Unlock(ctx, req.Passphrase, lock)
```

Comment:
so this will need more thought. yes you do need to have the wallet unlocked if changing an account to begin using a separate passphrase, but if simply changing the already-individually-encrypted account from an old to a new passphrase, that's not the case.

Reply:
That's a good point. I will address it